src/test/java/chapter5/Question2Test.java

```diff
@@ -0,0 +1,14 @@
+package chapter5;
+
+import org.junit.jupiter.api.Test;
+import src.chapter5.question2.Question2;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class Question2Test {
```

Comment:
😢 Method variations are needed. Try calling the method with a short and long string as well.

Reply:
Tests with method variations has been added.